### PR TITLE
#978 bugfixes

### PR DESCRIPTION
**Pull request by serkanxselcuk**
Bugfixes in issue #978 
------------
Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [. ] Provide a URL to a datasheet for the symbol(s) you are contributing
- [. ] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
- [. ] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required

### CODE REVIEW

File: Power_Protection.lib [hunk 1]
```diff
@@ -192,7 +192,6 @@ F2 "Package_TO_SOT_SMD:SOT-363_SC-70-6" 0 -450 50 H I C CNN
 F3 "" 80 75 50 H I C CNN
 $FPLIST
  *SC*70*
- *SC*88*
```
Comment (antoniovazquezblanco) [inline Power_Protection.lib]: It seems that NUP2202 is also available in SC-88 packaging.
Datasheet: https://www.onsemi.com/pub/Collateral/NUP2202W1-D.PDF
Maybe this component should have multiple filters but no default packaging. See KLC S5.1 "For generic symbols (those which map to multiple possible footprints), the Footprint field must be left blank".

What do you think?

Comment (serkanxselcuk) [reply Power_Protection.lib]: On [Datasheet](https://www.onsemi.com/pub/Collateral/NUP2202W1-D.PDF) and (Package Size Comparison)[http://www.onsemi.com/site/cdroms/micropackages/pdf-docs/onsemi/SOT563%20footprint-prof%23B3380.pdf]  SC-77, SC-88, SOT363, US6, UMT6, S-Mini6 have same dimensions.

What do you think about to represent this in footprint library and rename footprint? @antoniovazquezblanco

Comment (antoniovazquezblanco) [reply Power_Protection.lib]: I am learning new things here :)

For that I would like to get the input of someone with more experience than me. Let's ping someone and get some feedback:
@poeschlr, @SchrodingersGat, @pointhi, @evanshultz ?

Comment (serkanxselcuk) [reply Power_Protection.lib]: Learning new thinks :) this why I like to be here. 6 eyes see better than 4, get this checked by others cannot harm.

Comment (serkanxselcuk) [reply Power_Protection.lib]: Datasheet Page 1:
image

Datasheet Page 8:
image

**Comment by evanshultz** [comment]
I believe I already caught all of these, plus more, at https://github.com/KiCad/kicad-symbols/pull/882.

**Comment by evanshultz** [comment]
Keeping the first package name in the footprint name seemed most logical to me so I did that and eliminated any footprint filter using a package name that wasn't at the front of our footprint name.

File: Power_Protection.lib (hunk 1 above)
Comment (antoniovazquezblanco) [reply Power_Protection.lib]: Thank you for teaching me this stuff. You are totally right :).

**Comment by antoniovazquezblanco** [comment]
It seems that the work from @evanshultz was there before this PR so I will close this in favor of #882.

Thank you anyways @serkanxselcuk

**Pull request closed without merging**